plugins/woocommerce-admin/client/launch-your-store/settings/slotfill.js

```diff
@@ -183,20 +193,13 @@ const SiteVisibility = () => {
 						label={
 							<>
 								{ __(
-									'Restrict to store pages only',
+									'Apply to store pages only',
 									'woocommerce'
 								) }
 								<p>
-									{ createInterpolateElement(
-										__(
-											'Display a "coming soon" message on your <a>store pages</a> — the rest of your site will remain visible.',
-											'woocommerce'
-										),
-										{
-											a: createElement( 'a', {
-												href: SITE_VISIBILITY_DOC_LINK,
-											} ),
-										}
+									{ __(
+										'Display a “coming soon” message on your store pages — the rest of your site will remain visible.',
```

Comment:
Should we use `"` or `“` and `”` for the quotes? I see that the original string uses `"`?  It seems that we're using both of them in the codebase. 😅

Reply:
@chihsuan Yeah I assumed the design was inaccurate (clarified it in Figma as well). Some discussion regarding updating it here p1724125186383239/1724116284.730329-slack-C01SFMVEYAK